backend/api/Areas/Property/Controllers/ParcelController.cs

```diff
@@ -78,6 +78,22 @@ public IActionResult IsPidAvailable(int parcelId, int pid)
             return new JsonResult(result);
         }
 
+        /// <summary>
+        /// Check if PIN is available
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("check/pin-available")]
+        [HasPermission(Permissions.PropertyView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(Model.CheckPidAvailabilityResponseModel), 200)]
+        [SwaggerOperation(Tags = new[] { "parcel" })]
+        public IActionResult IsPinAvailable(int parcelId, int pin)
```

Comment:
If you're trying to determine if a PIN is available why do you ask for the `parcelId`?

Reply:
This is the `id `not the actual `pid`. It is the same way we check for the duplicate `pid` as well. It sends the `parcelId` and `pid`.  This also ensures that if it is the same parcel it finds the duplicate on it will not cause the error

Reply:
Or am I confusing myself

Reply:
Ah I see, misread that.  Thanks!

Reply:
I may need to double check the constraints in the DB on this.  I'm wondering if it's possible to have duplicate PIN if the PID is different.